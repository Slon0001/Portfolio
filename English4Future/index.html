<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>EnglishBoom. Бесплатный интенсив</title>
	<meta name="description" content="EnglishBoom. Бесплатный интенсив. Алена Привиденцева English4Future">
	<link rel="stylesheet" href="bootstrap/dist/css/bootstrap.min.css">
	<link href="https://multimedia.getresponse.com/102/16617102/photos/207000402.png" rel="shortcut icon">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/adaptive.css">
	<link href="css/animate.css" rel="stylesheet" type="text/css" media="all" />
	<link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/jquery.slick/1.6.0/slick.css"/>
	<link href='http://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500italic,700,500,700italic,900,900italic' rel='stylesheet' type='text/css'>
</head>
<body>

<!-- PRELOADER -->
<div id="preloader">
	<div id="status"></div>
</div>
<!-- //PRELOADER -->
<div class="preloader_hide"></div>

<div id="page">

<section class="wrap">
	<section class="header col-xs-12 col-md-12 col-sm-12 col-lg-12">
			<div class="logo">
				<img src="img/176341302.png" alt="">
			</div>
		<div class="container adapt clearfix">
			<div class="title1">
				<img src="img/450662002.png" alt="">
			</div>
			<div class="title2">
				<div>
					<img src="img/312481702.png" alt="">
					<p class="title_start">Онлайн Интенсив Для Начинающего И Среднего Уровня</p>
				</div>
				<p class="start">Старт 8 августа</p>
				<p class="start">Участие <span>бесплатно</span></p>
			</div>
		</div>
	</section>

	<section id="get_st" class="form col-xs-12 col-md-12 col-sm-12 col-lg-12">
		<div class="container adapt_teach">
			<p>Для того чтобы Принять Участие в Интенсиве, <br><b>Заполни эту Форму</b></p>
			<img src="img/450753102.png" alt="" class="img_form">
				<form action="" id="registr" class="col-xs-6 col-md-8 col-sm-6 col-lg-8 col-ss-12">
					<input type="text" class="name" value="Ваше имя">
					<input type="text" class="name" value="Ваш email">
					<button class="btn_form">Участвовать</button>
				</form>
		</div>
	</section>

	<section class="get_stake col-xs-12 col-md-12 col-sm-12 col-lg-12">
			<h2>Как все будет проходить?</h2>
		<div class="container_for_tasks">
				<ul class="tasks col-xs-12 col-md-12 col-sm-12 col-lg-12">
					<li class="tasks_item col-xs-6 col-md-3 col-sm-6 col-lg-3 col-ss-12 ">
						<img src="img/451516902.png" alt="">
						<p>Оставляешь заявку</p>
						<img src="img/365227702.png" class="right" alt="">
						
					</li>
					<li class="tasks_item col-xs-6 col-md-3 col-sm-6 col-lg-3 col-ss-12">
						<img src="img/451521302.png" alt="">
						<p>Получаешь доступ к интенсиву</p>
						<img src="img/365227702.png" class="right" alt="">
					</li>
					<li class="tasks_item col-xs-6 col-md-3 col-sm-6 col-lg-3 col-ss-12">
						<img src="img/451517102.png" alt="">
						<p>В течении недели получаешь ежедневные видео-уроки</p>
						<img src="img/365227702.png" class="right" alt="">
					</li>
					<li class="tasks_item col-xs-6 col-md-3 col-sm-6 col-lg-3 col-ss-12">
						<img src="img/451520302.png" alt="">
						<p>8 августа старт программы</p>
					</li>
				</ul>
				<nav class="navmenu">
					<ul>
						<li class="first scroll_btn"><a href="#get_st" class="btn_for_tasks" >Принять Участие</a></li>
					</ul>
				</nav>
		</div>
	</section>

	<section class="skill col-xs-12 col-md-12 col-sm-12 col-lg-12">
		<h2>ТВОЙ РЕЗУЛЬТАТ ПОСЛЕ ИНТЕНСИВА</h2>
			<div class="container">
				<div class="col-xs-7 col-md-7 col-sm-7 col-lg-7">
					<img src="img/451496002.png" alt="">
				</div>
				<div class="skill_text">
					<p>- Пополнишь свой словарный запас;</p>
					<p>- Научишься задавать вопросы по-английски и отвечать на них;</p>
					<p>- Начнешь воспринимать английскую речь на слух;</p>
					<p>- Поймешь что изучать английский можно в легкой и увлекательной форме.</p>
				</div>
			</div>
	</section>

	<section class="teacher col-xs-12 col-md-12 col-sm-12 col-lg-12">
		<h2>Как все будет проходить?</h2>
			<div class="container adapt_teach">
				<div class="col-xs-12 col-md-12 col-sm-12 col-lg-12">
					<img src="img/379847102.png" alt="" class="teacher_img">
				</div>
				<div class="teacher_text">
					<h3><b>Алена Привиденцева</b></h3>
					<p>- Основатель компании "English4Future";</p>
					<p>- Автор обучающих курсов: "Английская грамматика: Руководство", "Чемпионат. Разговорный английский за 30 дней", "English Barrier", "Upgrade";</p>
					<p>- Преподает английский язык более 7 лет;</p>
					<p>- Более 3500 выпускников по английскому языку в странах СНГ;</p>
					<p>- Автор книги: "Английская грамматика от А до Я", "Как разобраться во временах английского языка за 3 простых шага.</p>
				</div>
			</div>
	</section>

	<section class="footer">
		<div class="container">
			<div class="col-xs-12 col-md-12 col-sm-12 col-lg-12">
				<label for="ch" class="btn_footer">Участвовать бесплатно</label>
			</div>
		</div>
		<footer class="">
			<div class="container">
				<div class="contacts ">
					<img src="img/176341302.png" alt="">
					<span>По всем вопросам: info@english4future.net, +38 067 343 84 32 (Украина), +7 499 322 94 96 (Россия).ЧП Привиденцева Е.А. ИНН 3327909087</span>
					<a href="http://conf.english4future.net/"><p>Политика конфиденциальности</p></a>
				</div>
			</div>
		</footer>
	</section>
	<input type="checkbox" id="ch" class="checkbox-btn">
	<div class="open">
		<form  class="open_form">
			<input type="text" class="name" value="Ваше имя">
			<input type="text" class="name" value="Ваш email">
			<button class="btn_form be">Участвовать</button>
			<label class="btn_form" for="ch">Закрыть</label>
		</form>
	</div>
</section>
</div>

<!-- SCRIPTS -->
	<!--[if IE]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
    <!--[if IE]><html class="ie" lang="en"> <![endif]-->

	<script src="js/jquery.min.js" type="text/javascript"></script>
	<script src="js/jquery.BlackAndWhite.js"></script>
	<script src="js/bootstrap.min.js" type="text/javascript"></script>
	<script src="js/superfish.min.js" type="text/javascript"></script>
	<script src="js/animate.js" type="text/javascript"></script>
	<script src="js/myscript.js" type="text/javascript"></script>
	<script src="js/custom.js" type="text/javascript"></script>
	<script src="js/jquery.nicescroll.min.js" type="text/javascript"></script>
	
</body>
</html>