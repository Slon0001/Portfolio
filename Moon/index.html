<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>Moon</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
	
	<link rel="shortcut icon" href="images/favicon.png">
    
	<!-- CSS -->
	<link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" />
	<link rel="stylesheet" href="font-awesome-4.6.3/css/font-awesome.min.css">
	<link href="css/flexslider.css" rel="stylesheet" type="text/css" />
	<link href="css/prettyPhoto.css" rel="stylesheet" type="text/css" />
	<link href="css/animate.css" rel="stylesheet" type="text/css" media="all" />
    <link href="css/owl.carousel.css" rel="stylesheet">
	<link href="css/style.css" rel="stylesheet" type="text/css" />
    
	<!-- FONTS -->
	<link href='https://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>	
    
</head>
<body>
<!-- PRELOADER -->
<img id="preloader" src="https://d13yacurqjgara.cloudfront.net/users/99875/screenshots/1446794/gravity_drib.gif" alt="" />
<!-- //PRELOADER -->
<div class="preloader_hide">

	<div class="part js">

	<!-- PAGE -->
	<div id="page">

		<!-- HEADER -->
		<header>
			
			<!-- MENU BLOCK -->
			<div class="menu_block">
			
				<!-- CONTAINER -->
				<div class="container clearfix">
					
					<!-- LOGO -->
					<div class="logo pull-left">
						<a href="index.html" ><span class="b1">m</span><span class="b2">o</span><span class="b3">o</span><span class="b4">n</span></a>
					</div><!-- //LOGO -->
					
					<!-- SEARCH FORM -->
					<div id="search-form" class="pull-right">
						<form method="get" action="#">
							<input type="text" name="Search" value="Search" onFocus="if (this.value == 'Search') this.value = '';" onBlur="if (this.value == '') this.value = 'Search';" />
						</form>
					</div><!-- SEARCH FORM -->
					
					<!-- MENU -->
					<div class="pull-right">
						<nav class="navmenu center">
							<ul>
								<li class="first active scroll_btn"><a href="#home" >Преимущества</a></li>
								<li class="scroll_btn"><a href="#rates">Тарифы</a></li>
								<li class="scroll_btn"><a href="#projects">Калькулятор стоимости</a></li>
								<li class="scroll_btn"><a href="#team">Контакты</a></li>
							</ul>
						</nav>
					</div><!-- //MENU -->
				</div><!-- //MENU BLOCK -->
			</div><!-- //CONTAINER -->
		</header><!-- //HEADER -->
		
		
		<!-- HOME -->
		<section id="home" class="padbot0">
				
			<!-- TOP SLIDER -->
			<div class="flexslider top_slider">
				<ul class="slides">
					<li class="slide1">
						<div class="flex_caption1">
							<p class="title4 captionDelay7 FromBottom">Подруге муж купил Bentley, нечем крыть?</p>
							<p class="title1 captionDelay2 FromTop">Арендуй&nbsp</p>
							<p class="title2 captionDelay4 FromTop">участок&nbsp</p>
							<p class="title3 captionDelay6 FromTop"> на Луне!</p>
							<p class="title4 captionDelay7 FromBottom">И тебе непременно понадобится звездолёт, а он круче Bentley! Мы предлагаем не простую аренду земельных участков, мы предлагаем арендовать землю на Луне, только послушай, как это звучит.</p>
							<p class="title4 captionDelay7 FromBottom">Ты сможешь воспользоваться услугой посуточной аренды, пожизненной аренды и арендовать участок по пост оплате.</p>
						</div>
						<a class="slide_btn FromRight" href="javascript:void(0);" >Скорее к тарифам</a>
					<li class="slide2">
						<div class="flex_caption1">
							<p class="title1 captionDelay6 FromLeft">Арендуй</p>
							<p class="title2 captionDelay4 FromLeft">участок</p>
							<p class="title3 captionDelay2 FromLeft">на Луне!</p>
							<p class="title4 captionDelay7 FromLeft">И тебе непременно понадобится звездолёт, а он круче Bentley! Мы предлагаем не простую аренду земельных участков, мы предлагаем арендовать землю на Луне, только послушай, как это звучит.</p>
							<p class="title4 captionDelay7 FromBottom">Ты сможешь воспользоваться услугой посуточной аренды, пожизненной аренды и арендовать участок по пост оплате.</p>
						</div>
						<a class="slide_btn FromRight" href="javascript:void(0);" >Скорее к тарифам</a>
					</li>
					<li class="slide3">
						<div class="flex_caption1">
							<p class="title1 captionDelay1 FromBottom">Арендуй</p>
							<p class="title2 captionDelay2 FromBottom">участок</p>
							<p class="title3 captionDelay3 FromBottom">на Луне!</p>
							<p class="title4 captionDelay5 FromBottom">И тебе непременно понадобится звездолёт, а он круче Bentley! Мы предлагаем не простую аренду земельных участков, мы предлагаем арендовать землю на Луне, только послушай, как это звучит.</p>
							<p class="title4 captionDelay7 FromBottom">Ты сможешь воспользоваться услугой посуточной аренды, пожизненной аренды и арендовать участок по пост оплате.</p>
						</div>
						<a class="slide_btn FromRight" href="javascript:void(0);" >Скорее к тарифам</a>
					</li>
				</ul>
			</div>
		</section><!-- //HOME -->
		
		
		<!-- ABOUT -->
		<section id="about">
			
			<!-- SERVICES -->
			<div class="services_block padbot40" data-appear-top-offset="-200" data-animated="fadeInUp">
				
				<!-- CONTAINER -->
				<div class="container">
				
					<!-- ROW -->
					<div class="row">
					<div class="Prev_Benefits">
						<p class="prev_benefit_1">Не веришь, что на Луне возможно жить?</p>
						<p class="prev_benefit_2">Ты просто не знаешь всех преимуществ!</p>
					</div>
						<div class="col-lg-3 col-md-3 col-sm-6 col-xs-6 col-ss-12 margbot30">
							<i class="icon fa fa-sun-o" aria-hidden="true"></i>
							<a class="services_item" href="javascript:void(0);" >
								<p>На Луне очень тепло</p>
								<span>Луна-очень близкий к Солнцу спутник планеты.</span>
							</a>
						</div>
						<div class="col-lg-3 col-md-3 col-sm-6 col-xs-6 col-ss-12 margbot30">
							<i class="icon fa fa-users" aria-hidden="true"></i>
							<a class="services_item" href="javascript:void(0);" >
								<p>Никаких назойливых соседей</p>
								<span>Все участки расположены на огромном расстоянии друг от друга, ты сможешь шуметь сколько захочешь.</span>
							</a>
						</div>
						<div class="col-lg-3 col-md-3 col-sm-6 col-xs-6 col-ss-12 margbot30">
							<i class="icon fa fa-gift" aria-hidden="true"></i>
							<a class="services_item" href="javascript:void(0);" >
								<p>Крутой скафандра в <b>подарок!</b></p>
								<span>После подписания договора аренды ты получишь скафандр на срок аренды абсолютно бесплатно!.</span>
							</a>
						</div>
						<div class="col-lg-3 col-md-3 col-sm-6 col-xs-6 col-ss-12 margbot30">
							<i class="icon fa fa-bed" aria-hidden="true"></i>
							<a class="services_item" href="javascript:void(0);" >
								<p>НЕТ сонному состоянию</p>
								<span>На Луне нет атмосферы, поэтому там не бывает сумерек и ночь(или день) наступают мгновенно.</span>
							</a>
						</div>
					</div><!-- //ROW -->
					<div id="particles">
					<div class="after_benefit">
						<p class="prev_benefit_1 crct">Ну что? <b>Сомнения развеяны?</b></p>
						<a class="btn_benefit" href="javascript:void(0);" >Заказывай звонок менеджера</a>
					</div>
					</div>
				</div><!-- //CONTAINER -->
			</div><!-- //SERVICES -->
			
			<!-- CLEAN CODE -->

			<div class="cleancode_block">
				
				<!-- CONTAINER -->
				<div class="container" id="rates" data-appear-top-offset="-200" data-animated="fadeInUp">
					
					<!-- CASTOM TAB -->
					<div id="myTabContent" class="tab-content">
						<div class="tab-pane fade in active clearfix" id="tab1">
							<p>Тариф</p>
							<p class="title"><b>На</b> Сутки</p>
							<ul class="prev_rates">
								<li class="rates">Аренда на год до 13 М2;</li>
								<li class="rates">Предоплата 100%;</li>
								<li class="rates">Скафандр "не очень" в подарок.</li>
							</ul>
							<p class="title">от <b>450 000</b> руб/мес.</p>
							<a class="btn_benefit" href="javascript:void(0);" >Заказать</a>
							<!-- Copy from Benefit -->
						</div>
						<div class="tab-pane fade clearfix" id="tab2">
							<p>Тариф</p>
							<p class="title"><b>Пост</b> оплата</p>
							<ul class="prev_rates">
								<li class="rates">Аренда на год до 40 М2;</li>
								<li class="rates">Пост оплата;</li>
								<li class="rates">Скафандр "норм" в подарок;</li>
								<li class="rates">Возможность выбора геометрической формы участка.</li>
							</ul>
							<p class="title">от <b>680 000</b> руб/мес.</p>
							<a class="btn_benefit" href="javascript:void(0);" >Заказать</a>
							<!-- Copy from Benefit -->
						</div>
						<div class="tab-pane fade clearfix" id="tab3">
							<p>Тариф</p>
							<p class="title"><b>Пожизненный</b></p>
							<ul class="prev_rates">
								<li class="rates">Аренда на 99 лет до 2300 М2;</li>
								<li class="rates">Предоплата 100%;</li>
								<li class="rates">Скафандр "лакшери" в подарок;</li>
								<li class="rates">Возможность выбора геометрической формы участка;</li>
								<li class="rates">Участок расположен в лучшем районе Луны.</li>
							</ul>
							<p class="title">от <b>12 900 000</b> руб/мес.</p>
							<a class="btn_benefit" href="javascript:void(0);" >Заказать</a>
							<!-- Copy from Benefit -->
						</div>
						<div class="tab-pane fade clearfix" id="tab4">
							<p>Тариф</p>
							<p class="title">Свой<b>Собственный</b></p>
							<ul class="prev_rates">
								<li class="rates">На какой срок аредновать выбираешь сам;</li>
								<li class="rates">Предоплата или пост оплата;</li>
								<li class="rates">Скафандр в подарок на выбор;</li>
								<li class="rates">Возможность выбора геометрической формы участка;</li>
								<li class="rates">Выбирай район, в котором расположен участок;</li>
								<li class="rates">Стоимость зависит полностью от твоего выбора.</li>
							</ul>
							<a class="btn_benefit" href="javascript:void(0);" >Создать</a>
							<!-- Copy from Benefit -->
						</div>
					</div>
					<ul id="myTab" class="nav nav-tabs">
						<li class="active"><a class="i1" href="#tab1" data-toggle="tab" ><i></i><span>На Сутки</span></a></li>
						<li><a class="i2" href="#tab2" data-toggle="tab" ><i></i><span>Пост оплата</span></a></li>
						<li><a class="i3" href="#tab3" data-toggle="tab" ><i></i><span>Пожизненный</span></a></li>
						<li><a class="i4" href="#tab4" data-toggle="tab" ><i></i><span>Свой собственный</span></a></li>
					</ul><!-- CASTOM TAB -->
				</div><!-- //CONTAINER -->
			</div><!-- //CLEAN CODE -->
			</div>
		</section>
	</div><!-- //PAGE -->

	
	<!-- CONTACTS -->
	<section id="contacts">
	</section><!-- //CONTACTS -->
	
	<!-- FOOTER -->
	<footer>
			
		<!-- CONTAINER -->
		<div class="container">
			
			<!-- ROW -->
			<div class="row cont_foot" data-appear-top-offset="-200" data-animated="fadeInUp">
				
				<div class="col-lg-4 col-md-4 col-sm-6 padbot30">
					<p class="foot_text">
						Для того, чтобы арендовать участок прямо сейчас, введи свои данные в форму справа.
					</p>
				</div>
					
					<!-- CONTACT FORM -->
					<div class="col-lg-4 col-md-4 col-sm-6 padbot30">
					<div class="span9 contact_form">
						<div id="note"></div>
						<div id="fields">
							<form id="contact-form-face" class="clearfix" action="#">
								<input type="text" name="name" value="Name" onFocus="if (this.value == 'Name') this.value = '';" onBlur="if (this.value == '') this.value = 'Name';" />
								<textarea name="message" onFocus="if (this.value == 'Message') this.value = '';" onBlur="if (this.value == '') this.value = 'Message';">Message</textarea>
								<input class="contact_btn" type="submit" value="Send message" />
							</form>
						</div>
					</div><!-- //CONTACT FORM -->
					</div>
				</div>
			</div><!-- //ROW -->
		</div><!-- //CONTAINER -->
	</footer><!-- //FOOTER -->

<!-- SCRIPTS -->
	<!--[if IE]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
    <!--[if IE]><html class="ie" lang="en"> <![endif]-->
	
	<script src="js/jquery.min.js" type="text/javascript"></script>
	<script src="js/jquery.prettyPhoto.js" type="text/javascript"></script>
	<script src="js/jquery.nicescroll.min.js" type="text/javascript"></script>
	<script src="js/jquery.flexslider-min.js" type="text/javascript"></script>
	<script src="js/jquery.BlackAndWhite.js"></script>
	<script src="js/bootstrap.min.js" type="text/javascript"></script>
	<script src="js/superfish.min.js" type="text/javascript"></script>
	<script src="js/owl.carousel.js" type="text/javascript"></script>
	<script src="js/animate.js" type="text/javascript"></script>
	<script src="js/myscript.js" type="text/javascript"></script>
	<script type='text/javascript' src='js/jquery.particleground.min.js'></script>
</body>
</html>