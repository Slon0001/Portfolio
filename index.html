<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="description" content="particles.js is a lightweight JavaScript library for creating particles.">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <title>Portfolio</title>
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/particles.css">
    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
</head>
<body> 
<div id="particles-js" class="particles"></div>
        <header class="header">
            <div class="hp-container">
                <h1 class="hp-text">
                    My Portfolio.
                </h1>
            </div>
        </header>

        <main class="main">
            <ul class="list portfolio">
                <li class="list__item">
                    <a class="block-link" href="diploms/Diplom/index.html">
                        <img src="images/Diplom.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Golden/index.html">
                        <img src="images/Golden.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Moon/index.html">
                        <img src="images/Moon.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Cheshirskaya/index.html">
                        <img src="images/Chehirskaya.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Take/index.html">
                        <img src="images/Take.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Hybric/index.html">
                        <img src="images/Hybric.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Photographer/index.html">
                        <img src="images/Photographer.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/English4Future/index.html">
                        <img src="images/English4Future.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/Flawless email/index.html">
                        <img src="images/mail.png" alt="">
                    </a>
                </li>
                <li class="list__item">
                    <a class="block-link" href="diploms/bg/index.html">
                        <img src="images/CSS3BG.png" alt="">
                    </a>
                </li>
            </ul>
        </main>

<!-- scripts -->
<script src="js/particles.js"></script>
<script src="js/app.js"></script>

</body>
</html>