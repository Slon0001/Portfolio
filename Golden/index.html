<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width; initial-scale=1.0">
	<title>Base</title>
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/base.css">
	<link rel="stylesheet" href="css/fonts.css">
	<link href='https://fonts.googleapis.com/css?family=Raleway:500' rel='stylesheet' type='text/css'>
	<link href='https://fonts.googleapis.com/css?family=Droid+Serif' rel='stylesheet' type='text/css'>
	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
	<link href='https://fonts.googleapis.com/css?family=Roboto+Slab' rel='stylesheet' type='text/css'>
	<link rel="stylesheet" media="screen and (max-width:960px)" href=""></head>
<body>
	<div class="new">
		<div class="wrap">
			<header class="header">
				<div class="header-top">
					<img class="logo" src="image/Golden_logo.png" alt="">
					<nav class="menu clearfix">
						<a href="" class="menu-item">Home</a>
						<a href="" class="menu-item">Services</a>
						<a href="" class="menu-item">Portfolio</a>
						<a href="" class="menu-item">About</a>
						<a href="" class="menu-item">Contact</a> <i class="icon-menu"></i> <i class="icon-search"></i>
					</nav>
				</div>
				<figure class="slide">
					<section class="title_inside_slide">
						<h2 class="welcome">Welcome To Our Studio!</h2>
						<h1 class="NTMU">It’s nice to Meet you</h1>
					</section>
				<button class="tell_me">
					<p>Tell Me More</p>
				</button>
				</figure>
			</header>
		</div>
		<div class="services">
			<h2 class="title-services">Services</h2>
			<p class="sub-title">Proin iaculis purus consequat sem cure.</p>
			<div class="cont">
				<div class="commerce">
					<li class="background">
						<img src="pic/icon-basket.png" alt="" class="pic_services-basket"></li>
					<h3 class="title-cws">E-Commerce</h3>
					<p class="description">
						Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="web">
					<li class="background">
						<img src="pic/laptop.png" alt="" class="pic_services-laptop"></li>
					<h3 class="title-cws">Responsive Web</h3>
					<p class="description">
						Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="security">
					<li class="background">
						<img src="pic/lock.png" alt="" class="pic_services-lock"></li>
					<h3 class="title-cws">Web Security</h3>
					<p class="description">
						Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
			</div>
		</div>
		<div class="gallery_inside clearfix">
			<h2 class="title-portfolio">Our portfolio</h2>
			<p class="sub-title">Proin iaculis purus consequat sem cure.</p>
			<ul class="prtflio clearfix">
				<li class="prtflio_item">
					<img src="image/brand.png" alt="" class="prtflio_pic">
					<h3 class="prtflio-title">Ebony & Ivory</h3>
					<p class="prtflio_sub-title">Branding</p>
				</li>
				<li class="prtflio_item">
					<img src="image/smart.png" alt="" class="prtflio_pic">
					<h4 class="prtflio-title">Smart Stationary</h4>
					<p class="prtflio_sub-title">Print Design</p>
				</li>
				<li class="prtflio_item">
					<img src="image/poster.png" alt="" class="prtflio_pic">
					<h3 class="prtflio-title">Clever Poster</h3>
					<p class="prtflio_sub-title">Print Design</p>
				</li>
				<li class="prtflio_item">
					<img src="image/music.png" alt="" class="prtflio_pic">
					<h3 class="prtflio-title">Vinyl Record</h3>
					<p class="prtflio_sub-title">Product Mock-Up</p>
				</li>
				<li class="prtflio_item">
					<img src="image/design.png" alt="" class="prtflio_pic">
					<h3 class="prtflio-title">Treehouse Template</h3>
					<p class="prtflio_sub-title">Web Design</p>
				</li>
				<li class="prtflio_item">
					<img src="image/logo.png" alt="" class="prtflio_pic">
					<h3 class="prtflio-title">Burned Logo</h3>
					<p class="prtflio_sub-title">Branding</p>
				</li>
			</ul>
		</div>
		<div class="container">
			<h3 class="about_title">About Us</h3>
			<p class="sub-title">Proin iaculis purus consequat sem cure.</p>
			<ul class="about">
				<li class="about_item">
					<div class="about_text">
						<h3>July 2010</h3>
						<h4>Our Humble Beginnings</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
						</p>
					</div>
					<figure class="about_img">
						<img src="image/krug1.jpg" alt=""></figure>
				</li>
				<li class="about_item">
					<div class="about_text">
						<h3>January 2011</h3>
						<h4>Facing Startup Battles</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
					<figure class="about_img">
						<img src="image/krug2.jpg" alt=""></figure>
				</li>
				<li class="about_item">
					<div class="about_text">
						<h3>December 2012</h3>
						<h4>Enter The Dark Days</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.
						</p>
					</div>
					<figure class="about_img">
						<img src="image/krug3.jpg" alt=""></figure>
				</li>
				<li class="about_item">
					<div class="about_text">
						<h3>february 2014</h3>
						<h4>Our Triumph</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
						</p>
					</div>
					<figure class="about_img">
						<img src="image/krug4.jpg" alt=""></figure>
				</li>
				<button class="about_button">
					<h4 class="text_for_btn">Our Story Continues ...</h4>
				</button>
			</ul>
		</div>
		<div class="block">
			<h3 class="title_team">our amazing team</h3>
			<p class="sub-title">Proin iaculis purus consequat sem cure.</p>
			<div class="contain">
				<div class="step1">
					<img src="image/container.jpg" alt="" class="black">
					<h4 class="initials">Kimberly Thompson</h4>
					<p class="initials_">Marketer</p>
					<div class="ic">
						<li class="icons">
							<img src="pic/twitter.png" alt="" class="tw"></li>
						<li class="icons">
							<img src="pic/facebook.png" alt="" class="fb"></li>
						<li class="icons">
							<img src="pic/pinterest.png" alt="" class="pin"></li>
					</div>
				</div>
				<div class="step1">
					<img src="image/container.jpg" alt="" class="black">
					<h4 class="initials">Rico Massimo</h4>
					<p class="initials_">Coder</p>
					<div class="ic">
						<li class="icons">
							<img src="pic/twitter.png" alt="" class="tw"></li>
						<li class="icons">
							<img src="pic/facebook.png" alt="" class="fb"></li>
						<li class="icons">
							<img src="pic/google.png" alt="" class="gog"></li>
					</div>
				</div>
				<div class="step1">
					<img src="image/container.jpg" alt="" class="black">
					<h4 class="initials">Uku Mason</h4>
					<p class="initials_">Graphic Designer</p>
					<div class="ic_s">
						<li class="icons">
							<img src="pic/twitter.png" alt="" class="tw"></li>
						<li class="icons">
							<img src="pic/pinterest.png" alt="" class="pin"></li>
					</div>
				</div>
				<h4 class="descr">
					Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  
					aenean rhoncus posuere odio in tincidunt proin iaculis.
				</h4>
			</div>
		</div>
		<div class="footer_head">
			<ul class="media">
				<li class="media_img">
					<img src="image/logo1.jpg" alt="" class="correct"></li>
				<li class="media_img">
					<img src="image/logo2.jpg" alt=""></li>
				<li class="media_img">
					<img src="image/logo3.jpg" alt=""></li>
				<li class="media_img">
					<img src="image/logo4.jpg" alt=""></li>
			</ul>
		</div>
		<div class="footer_content">
			<div class="form_content clearfix">
				<h3 class="title_footer">Contact us</h3>
				<h4 class="sub-title">Proin iaculis purus consequat sem cure.</h4>
				<form class="form_input">
					<input type="text" name="name" placeholder="your name*" class="form1"/>
					<input type="text" name="name" placeholder="your e-mail*" class="form1"/>
					<input type="text" name="name" placeholder="subject" class="form1"/>
				</form>
				<form class="footer_textarea">
					<textarea name="name" placeholder="Your message*" cols="30" rows="9" class="form2"></textarea>
				</form>
			</div>
			<button class="tell_me">
				<p>Send Message</p>
			</button>
		</div>
		<div class="section_footer">
			<p class="text">© Copyright 2014 FreebiesXpress.com</p>
			<div class="social">
				<li class="ic-f">
					<img src="pic/twitter-w.png" alt="" class="tw-w"></li>
				<li class="ic-f">
					<img src="pic/facebook-w.png" alt="" class="fb-w"></li>
				<li class="ic-f">
					<img src="pic/pinterest-w.png" alt="" class="pin-w"></li>
				<li class="ic-f">
					<img src="pic/google-w.png" alt="" class="gog-w"></li>
			</div>
		</div>
	</div>
</body>
</html>